flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -112,7 +113,12 @@ class DataStreamOverAggregate(
               "condition.")
         }
       case _: RowTimeType =>
-        throw new TableException("OVER Window of the EventTime type is not currently supported.")
+        if (overWindow.lowerBound.isUnbounded && overWindow.upperBound.isCurrentRow) {
+          createUnboundedAndCurrentRowEventTimeOverWindow(inputDS)
```

Comment:
EVENT-TIME OVER Need to treatment "rows" and "range" clause separately, because 
-  ROWS specifies the window in physical units (rows).
-  RANGE specifies the window as a logical offset.

They have different semantics, for example:
DATA:
```
(long, int, String)
(1L, 1, "H")
(2L, 2, "H")
(2L, 3,"H")
```
ROWS sum(b) result: `1,3,6`
RANGE sum(b) result: `1,6,6`